services/src/test/scala/cromwell/pipeline/service/UserServiceTest.scala

```diff
@@ -55,35 +46,43 @@ class UserServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
     }
 
     "updateUser" should {
-      "returns success if database handles query" taggedAs Service in {
-        val numberId = "123"
-        val updatedUser =
-          dummyUser.copy(email = "updatedEmail", firstName = "updatedFirstName", lastName = "updatedLastName")
-        val request = UserUpdateRequest("updatedEmail", "updatedFirstName", "updatedLastName")
 
-        when(userRepository.getUserById(userId)).thenReturn(Future(Some(dummyUser)))
-        when(userRepository.updateUser(updatedUser)).thenReturn(Future.successful(1))
-
-        userService.updateUser(numberId, request).map { result =>
+      "returns success if database handles query" in {
+        val user = TestUserUtils.getDummyUser()
+        val user2 = user.copy(
+          email = UserEmail("updatedEmail@mail.com"),
+          firstName = Name("updatedFirstName"),
+          lastName = Name("updatedLastName")
+        )
+        val request = UserUpdateRequest(
+          user2.email,
+          user2.firstName,
+          user2.lastName
+        )
+
+        when(userRepository.getUserById(user.userId)).thenReturn(Future(Some(user)))
```

Comment:
Future.successful ?

Reply:
Nope

Reply:
Why not?

Reply:
Because method getUserById expects an Future[Option[User]] type, how to do otherwise?

Reply:
`Future.successful(Some(user)) `
This is the way?

Reply:
This is the way

Reply:
Done